dynamo/preprocessing/cell_cycle.py

```diff
@@ -418,7 +418,8 @@ def get_cell_phase(
     else:
         cell_phase_genes = gene_list
 
-    adata.uns["cell_phase_genes"] = cell_phase_genes
+    adata.uns["cell_phase_genes_order"] = [key for key in cell_phase_genes]
```

Comment:
the keys are just the phases, may change `cell_phase_genes_order` to `cell_phase_order`